mysqld_exporter.go

```diff
@@ -252,7 +252,10 @@ func main() {
 		http.Handle("/", landingPage)
 	}
 	http.HandleFunc("/probe", handleProbe(enabledScrapers, logger))
-
+	http.HandleFunc("/-/reload", func(w http.ResponseWriter, r *http.Request) {
+		c.ReloadConfig(*configMycnf, *mysqldAddress, *mysqldUser, *tlsInsecureSkipVerify, logger)
```

Comment:
`ReloadConfig` returns an error, we need to check this error and return an appropriate HTTP status code.